### ISSUE TITLE
Deploy to docs.opentermsarchive.org

### ISSUE THREAD

**Issue by MattiSG** [OpenTermsArchive/docs#2]
Reverse proxy opentermsarchive.org/doc to the GitHub Pages instance.

**Comment by MattiSG** [comment]
@Ndpnt if you have a clear idea of where the current proxy rules are set up, could you please take care of this? 🙂

**Comment by clementbiron** [comment]
They are in https://github.com/OpenTermsArchive/reverseproxy-ops

**Comment by clementbiron** [comment]
@Ndpnt suggest to use `docs.opentermarchive.org`(as docs.github.com docs.npmjs.com) and I find the proposal attractive.

**Comment by clementbiron** [comment]
@Ndpnt does not seem to have the time to setup proxy rules and i have no idea how to handle it.

What I tried on my side:
- configure the CNAME at gandi so that `docs.opentermsarchive.org` points to `opentermsarchive.github.io.`
- configure the custom domain github pages to `docs.opentermsarchive.org`

Which result on a accesible documentation website here [docs.opentermsarchive.org](docs.opentermsarchive.org) but:
- the Hugo config `baseURL`must be updated with `https://docs.opentermsarchive.org` value
-  Enforce `HTTPS` is not yet available, we need to wait for the TLS certificate is being provisioned

**Comment by MattiSG** [comment]
> suggest to use `docs.opentermarchive.org`

No problem on my side, that makes it easier to deploy, but that also makes it pretty much impossible to have a smooth progressive transition from https://github.com/ambanum/OpenTermsArchive.org to this stack, which was one of the ambitions.

**Comment by clementbiron** [comment]
Further to our discussion this morning, the site is live on https://docs.opentermsarchive.org/

**Comment by MattiSG** [comment]
If you believe the user value described in an issue is implemented, please close the issue 🙂 

In this case however, I always get a 404 on https://docs.opentermsarchive.org/, even after #14 has been merged, so there seems to be a problem 😉

**Comment by MattiSG** [comment]
Unfortunately still a 404 🙁

**Comment by MattiSG** [comment]
Indeed, when I look at the [`gh-pages`](https://github.com/OpenTermsArchive/docs/tree/gh-pages) branch, the `CNAME` file added in #15 is not present 🙂 in a similar fashion, each commit created through the GitHub settings ([example](https://github.com/OpenTermsArchive/docs/commit/7483b430b5a0fc34d8387f56c34d3faa8e99fc57)) is overridden on each deployment. The solution is thus to ensure the `CNAME` file is present in the generated website 🙂
